### PR TITLE
Log enhancements

### PR DESCRIPTION
**Pull request by ashcrow**
More information: https://docs.python.org/3/howto/logging.html#optimization

### CODE REVIEW

**Comment by ashcrow** [comment]
@mbarnes ^^

File: src/commissaire_http/server/cli.py [hunk 1]
```diff
@@ -91,8 +91,8 @@ def main():
 
         # Serve until we are killed off
         server.serve_forever()
-    except KeyboardInterrupt:  # pragma: no cover
-        pass
+    except KeyboardInterrupt as err:  # pragma: no cover
```
Comment (mbarnes) [inline src/commissaire_http/server/cli.py]: Was `as err` a leftover from earlier editing?  Looks not needed.

**Comment by ashcrow** [comment]
@mbarnes ⬆️

**Comment by mbarnes** [comment]
Squashed and merged manually.

**Pull request closed without merging**